### PR TITLE
Updating events module location for dbt 1.8

### PR DESCRIPTION
**Pull request by pgoslatara**
## What are you changing in this pull request and why?

dbt 1.8 changed the location of the `events` module from `dbt` to `dbt_common`. This PR updates the example accordingly.

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

**Review by nghi-ly — COMMENTED** [review]
thanks so much for opening this PR, @pgoslatara ! really appreciate your contribution! let me look into this for you.

**Comment by nghi-ly** [comment]
heya @dbeatty10, this edit looks correct to me.

before i approve/merge, do we need to use `VersionBlock` here for the example since it's specific to v1.8 ? i don't see it used on the page at all so i don't think i do but wanted to double-check with you first. thanks!

**Comment by dbeatty10** [comment]
> before i approve/merge, do we need to use `VersionBlock` here for the example since it's specific to v1.8 ? i don't see it used on the page at all so i don't think i do but wanted to double-check with you first. thanks!

Good catch @nghi-ly 🦅  -- indeed this does need different `VersionBlock`s for code this code example prior to v1.8 versus 1.8+.

None of the content on the page was valid until v1.5.

**Comment by nghi-ly** [comment]
@dbeatty10 : thanks for confirming this! will add versionblock

File: website/docs/reference/programmatic-invocations.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Register `callbacks` on dbt's `EventManager`, to access structured events and en
 
 ```python
 from dbt.cli.main import dbtRunner
-from dbt.events.base_types import EventMsg
+from dbt_common.events.base_types import EventMsg
```
Comment (nghi-ly) [inline website/docs/reference/programmatic-invocations.md:96]: hello @pgoslatara : looks like we'll need to have two copies of this python example. one for v1.5 - v.17 and the other for v1.8 and later.  

can you update your PR so it looks like the following:  

`<VersionBlock firstVersion="1.8">`

Copy the entire python code block here except use your dbt_common change

`</VersionBlock>`

`<VersionBlock lastVersion="1.7">`

Copy the entire python code block here, exactly how it looks right now

`</VersionBlock>`

Comment (nghi-ly) [reply website/docs/reference/programmatic-invocations.md:96]: @pgoslatara : you can click on this for an example containing the usage of `VersionBlock`: https://github.com/dbt-labs/docs.getdbt.com/blob/ccaf6b9877e378cde258955e27366d2b181ce9fd/website/docs/docs/core/pip-install.md?plain=1#L40

**Review by nghi-ly — COMMENTED** [review]
@pgoslatara : left one suggestion for you. i'm happy to make this change on your behalf as well, please let me know!

File: website/docs/reference/programmatic-invocations.md (hunk 1 above)
Comment (pgoslatara) [reply website/docs/reference/programmatic-invocations.md:96]: Thanks @nghi-ly for the helpful reference! Feedback implemented in e761a65.

**Review by nghi-ly — APPROVED** [review]
fantastic, @pgoslatara ! looks great!

**Pull request merged**